2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,29 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    bottle.volume = 100;
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    bottle.volume += 10;
+    if (bottle.volume > 100) {
+      bottle.volume = 100;
+    } 
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    bottle.volume -= 10;
+    if (bottle.volume < 0) {
+      bottle.volume = 0;
```

Comment:
You also need to use this method as well as a return because object methods acts like normal function